From 25238777efa842a3a83a47fad6abd45df1a09943 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Thu, 8 Oct 2020 16:06:34 -0700
Subject: [PATCH 067/322] [FIO toup] imx8mq_evk: set bootm len to 64

Allow larger bootm images (required by LmP).

Fixes:
   Uncompressing Kernel Image
Error: inflate() returned -5
Image too large: increase CONFIG_SYS_BOOTM_LEN
Must RESET board to recover

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/imx8mq_evk.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index 78390121113..fe2145bac00 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -180,6 +180,7 @@
 #define CONFIG_SYS_SDRAM_BASE           0x40000000
 #define PHYS_SDRAM                      0x40000000
 #define PHYS_SDRAM_SIZE			0xC0000000 /* 3GB DDR */
+#define CONFIG_SYS_BOOTM_LEN		SZ_64M
 
 #define CONFIG_MXC_UART_BASE		UART1_BASE_ADDR
 
-- 
2.34.1

